### PR TITLE
Migrate to 11ty 1.0

### LINKED ISSUE #705: [infra] update to 11ty 1.0.0
Update to 11ty 1.0 to prepare for Lit SSR

### PR DESCRIPTION
**Pull request by e111077**
fixes #705

### CODE REVIEW

File: packages/lit-dev-content/.eleventy.js [hunk 1]
```diff
@@ -42,6 +42,9 @@ const DEV = ENV.eleventyMode === 'dev';
 const cspInlineScriptHashes = new Set();
 
 module.exports = function (eleventyConfig) {
+  eleventyConfig.setLiquidOptions({
+    dynamicPartials: false, // fixes issues caused by MD id header syntax
```
[the remark this answers is not in the document]
Comment (aomarks) [reply packages/lit-dev-content/.eleventy.js:46]: Is there a different syntax we can use?

[the remark this answers is not in the document]
Comment (e111077) [reply packages/lit-dev-content/.eleventy.js:46]: Do you know what plugin handles this on the markdown-it side? I was having trouble pinning down where this was handled

[the remark this answers is not in the document]
Comment (aomarks) [reply packages/lit-dev-content/.eleventy.js:46]: Maybe markdownItAnchor?

[the remark this answers is not in the document]
Comment (e111077) [reply packages/lit-dev-content/.eleventy.js]: The issue was actually a red herring due to compilation making the error point to the wrong line. I fixed it. It was just that `{% import my-file.html $}` was missing quotes around the filename. All updated now

[the remark this answers is not in the document]
Comment (aomarks) [reply packages/lit-dev-content/.eleventy.js]: Nice! 💯

**Review by aomarks — APPROVED** [review]

**Comment by e111077** [comment]
integration test looks like a drop-shadow flake since it passed on the other run

**Comment by aomarks** [comment]
I just noticed a small regression from this upgrade.

It looks like our blog posts now always list "Lit Team" as an author. Previously, this was only the default.

Before: https://pr704-4653621---lit-dev-5ftespv5na-uc.a.run.app/blog/
After: https://pr708-0a5d38c---lit-dev-5ftespv5na-uc.a.run.app/blog/

Same issue on the blog posts themselves.

**Comment by e111077** [comment]
WE ARE ALL THE LIT TEAM

**Pull request merged**